src/recipe/DTOs/update-recipe.dto.ts

```diff
@@ -13,9 +14,10 @@ export class UpdateRecipeDTO {
   @ApiProperty({ example: 'Easy dumplings recipe', type: 'string' })
   description?: string;
 
-  @IsString()
-  @ApiProperty({ example: 'imageKey', type: 'string' })
-  imageKey?: string;
+  @ValidateNested()
+  @Type(() => CreateImageDTO)
+  @ApiProperty({ type: CreateImageDTO })
```

Comment:
you have marked this property as optional but the there is no `@IsOptional()` docorator